### PR TITLE
finished docs for LS-node

### PR DESCRIPTION
**Pull request by youngsun4786**

### CODE REVIEW

File: docs/getting-started/install.md [hunk 1]
```diff
@@ -23,69 +24,68 @@ A suitable python installation with `pip`. Minimum version requirement for Pytho
...
-    ```
-    
+
+   ```bash
+   cd studio
+   ```
+
 5. We highly recommend creating a Python virtual environment inside the `studio` folder by running the command:
-    
+
```
Comment (smahmed776) [inline docs/getting-started/install.md:79]: Env is not recommended for Windows users. As we don't have support for that yet in powershell script.

File: docs/getting-started/install.md [hunk 2]
```diff
@@ -23,69 +24,68 @@ A suitable python installation with `pip`. Minimum version requirement for Pytho
...
    python3.10 -m venv env
    ```
 
 6. Finally, run:
 
-    ```bash
-    .\flojoy -v env
-    ``` 
+   ```bash
+   .\flojoy -v env
```
Comment (smahmed776) [inline docs/getting-started/install.md:87]: Env is not supported by Windows Powershell script yet

**Review by smahmed776 — DISMISSED** [review]
Left a couple of comments.

**Comment by scorinaldi** [comment]
ok @youngsun4786 could you rebase this PR to resolve the conflicts, and then address the requested changes?

**Comment by youngsun4786** [comment]
> ok @youngsun4786 could you rebase this PR to resolve the conflicts, and then address the requested changes?

will look into it rn

**Review by scorinaldi — APPROVED** [review]
LGTM as per Syed's comments and my quick once-over

**Pull request merged**